app/components/SubmitColorsButton/SubmitColorsButton.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link, } from 'react-router-dom';
+
+export default function SubmitColorsButton() {
+  const linkProps = {
+    className: 'btn',
+    to: { pathname: '/', },
+    children: 'Submit new colors',
```

Comment:
UX: "Submit" is not a verb normal people use. When did you ever submit in real life last time? :)

Reply:
done